serialization/Payloads/sdoc-v1.0.json

```diff
@@ -0,0 +1,97 @@
+{
```

Comment:
This payload example does not validate against the schema (even after adding the "payload" key on top). It would be nice to have an example that does.

Most of the problems are related to lists like "element" listing only the elements' Ids, while the schema expects the full elements.

Reply:
Hmm - I'll investigate, I hadn't run into errors ...

The "element" property in all types (e.g. Sbom/element) must have ids because elements reference other elements.  The "element" property in Payload contains the full value for all elements in that payload.